File: PRAKTIKUM 5/PRAK504_2310817220028_SheilaSabina.py
def reverse(num):
    number = 0
    while num > 0:
        number = number * 10 + num % 10
        num //= 10
    return number

def main():
    print("Input")
    A, B = map(int, input().split())

    A = reverse(A)
    B = reverse(B)

    C = A + B
    result = reverse(C)

    print("\nOutput")
    print(result)

if __name__ == "__main__":
    main()

File: PRAKTIKUM 5/PRAK504_2310817220028_SheilaSabina.c
#include <stdio.h>
int reverse(int num) {

    int number = 0;
    while (num > 0) {
        number = number * 10 + num % 10;
        num /= 10;
    }
    return number;
}

int main() {
    int A, B;
    
    printf("Input\n");
    scanf("%d %d", &A, &B);
    
    A = reverse(A);
    B = reverse(B);
    
    int C = A + B;
    int result = reverse(C);
    
    printf("\nOutput\n");
    printf("%d", result);

    return 0;
}